doc/releases/changelog-dev.md

```diff
@@ -227,6 +227,8 @@ array([False, False])
   [(#4465)](https://github.com/PennyLaneAI/pennylane/pull/4465/)
   [(#4478)](https://github.com/PennyLaneAI/pennylane/pull/4478/)
 
+* `qml.ctrl(qml.PauliX)` returns a `Controlled` instead of a `MultiControlledX` like before.
```

Comment:
```suggestion
* `qml.ctrl(qml.PauliX)` returns a `Controlled` instead of a `MultiControlledX` like before.
  [(#4486)](https://github.com/PennyLaneAI/pennylane/pull/4486)
```